account-kit/signer/src/base.ts

```diff
@@ -831,6 +849,10 @@ export abstract class BaseAlchemySigner<TClient extends BaseSignerClient>
       });
     });
   };
+
+  private emitNewUserEvent = (isNewUser: boolean) => {
```

Comment:
re: the comments above

```suggestion
  private emitNewUserEvent = (isNewUser?: boolean) => {
```